styles.css

```diff
@@ -0,0 +1,136 @@
+:root {
+  --color-primary: hsl(0, 0%, 20%);
+  --color-primary-light: hsl(0, 0%, 30%);
+  --color-accent: #ff6b00;
+  --nav-height: 60px;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: gainsboro;
+  background-color: var(--color-primary-light);
+  position: fixed;
+  height: var(--nav-height);
+  top: 0;
+  left: 0;
+  width: 100%;
+}
+
+#logo {
+  margin-left: 10px;
+}
+
+#logo a {
+  color: white;
+  text-decoration: none;
+  font-weight: 800;
+}
+
+#nav-bar {
+  list-style: none;
+  padding: 0 20px;
+}
+
+#nav-bar li {
+  display: inline-block;
+}
+
+#nav-bar li a {
+  color: white;
+}
+
+#welcome-section {
+  background-color: gray;
+  background-color: var(--color-primary);
+  background-image: url("images/figure.png");
+  background-size: 60% auto;
+  background-repeat: no-repeat;
+  background-position: 150% 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: var(--nav-height);
+  height: 80vh;
```

Comment:
Great job with your styles 👏🏽. i  like it 👍🏽
unfortunately your section height doesn't match the Figma design

**Your  design**


<img width="282" alt="Screenshot 2022-06-30 at 02 07 36" src="https://user-images.githubusercontent.com/31113416/176564896-d55efcc4-c254-494c-b4bf-190fc5883364.png">

**Figma Design**
<img width="248" alt="Screenshot 2022-06-30 at 02 59 44" src="https://user-images.githubusercontent.com/31113416/176565469-1c7118a4-9e32-40fd-9ff0-f9e7fa1fb00f.png">


- [ ] Kindly make the Welcome section fit the entire screen height for mobile views.